### PR TITLE
[FLINK-17527][flink-dist] Make kubernetes-session.sh use session log4j/logback configuration files

### PR DESCRIPTION
**Pull request by wangyang0918**
## What is the purpose of the change

Currently, `kubernetes-session.sh` uses the log4j-console.properties and `yarn-session.sh` uses `log4j-yarn-session.properties`. They should have the same logging configuration. This PR will rename `log4j-yarn-session.properties` to `log4j-session.properties`, `logback-yarn.xml` to `logback-session.xml`. This also will make the logback configuration aligned with log4j configuration.

## Brief change log

* 21d91c1c0e4c75beebc9b032ca3106977f65b073 Rename yarn log4j/logback configuration files so that they could be reused
* c3caf6974bdf8298376d150b00fd7e7125089986 Make `kubernetes-session.sh` use session log4j/logback configuration files

## Verifying this change

* Manually test on a Yarn/K8s cluster, the client console logging should be same with before

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (**yes** / no / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)
  - If yes, how is the feature documented? (**not applicable** / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wangyang0918** [comment]
cc @tillrohrmann Could you please help to review this change?

File: docs/monitoring/logging.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Log4j2 is controlled using property files. In Flink's case, the file is usually
 Flink ships with the following default properties files:
 
 - `log4j-cli.properties`: Used by the Flink command line client (e.g. `flink run`) (not code executed on the cluster)
-- `log4j-yarn-session.properties`: Used by the Flink command line client when starting a YARN session (`yarn-session.sh`)
+- `log4j-session.properties`: Used by the Flink command line client when starting a YARN session (`yarn-session.sh`)
```
Comment (tillrohrmann) [inline docs/monitoring/logging.md:38]:
```suggestion
- `log4j-session.properties`: Used by the Flink command line client when starting a YARN or Kubernetes session (`yarn-session.sh`, `kubernetes-session.sh`)
```

**Comment by wangyang0918** [comment]
@tillrohrmann Thanks for the `fixup`. 

BTW, i do not suggest to backport this PR to release-1.10. Since we rename two files(e.g. `log4j-yarn-session.properties` -> `log4j-session.properties`, `logback-yarn.xml` -> `logback-session.xml`). Then it will be a surprise to our users with a `bugfix` upgrading.

**Comment by tillrohrmann** [comment]
Yes agreed @wangyang0918. It should only go into `1.11` and `1.12`.

**Pull request closed without merging**